test/ex_doc/refs_test.exs

```diff
@@ -26,6 +41,33 @@ defmodule ExDoc.RefsTest do
     assert Refs.get_visibility({:function, InMemory, :foo, 0}) == :public
     assert Refs.get_visibility({:function, InMemory, :foo, 9}) == :undefined
 
+    assert Refs.get_visibility({:function, WithModuleDoc, :foo, 1}) == :public
+    assert Refs.get_visibility({:function, WithModuleDoc, :_foo, 1}) == :hidden
+    assert Refs.get_visibility({:function, WithModuleDoc, :foo, 2}) == :hidden
+    assert Refs.get_visibility({:function, WithModuleDoc, :_foo, 2}) == :hidden
+    assert Refs.get_visibility({:function, WithModuleDoc, :foo, 3}) == :public
+    assert Refs.get_visibility({:function, WithModuleDoc, :_foo, 3}) == :public
+    assert Refs.get_visibility({:function, WithModuleDoc, :foo, 4}) == :undefined
+    assert Refs.get_visibility({:function, WithModuleDoc, :_foo, 4}) == :undefined
+
+    assert Refs.get_visibility({:function, WithoutModuleDoc, :foo, 1}) == :public
+    assert Refs.get_visibility({:function, WithoutModuleDoc, :_foo, 1}) == :hidden
+    assert Refs.get_visibility({:function, WithoutModuleDoc, :foo, 2}) == :hidden
+    assert Refs.get_visibility({:function, WithoutModuleDoc, :_foo, 2}) == :hidden
+    assert Refs.get_visibility({:function, WithoutModuleDoc, :foo, 3}) == :public
+    assert Refs.get_visibility({:function, WithoutModuleDoc, :_foo, 3}) == :public
+    assert Refs.get_visibility({:function, WithoutModuleDoc, :foo, 4}) == :undefined
+    assert Refs.get_visibility({:function, WithoutModuleDoc, :_foo, 4}) == :undefined
+
+    assert Refs.get_visibility({:function, InMemory, :foo, 1}) == :public
+    # assert Refs.get_visibility({:function, InMemory, :_foo, 1}) == :hidden
```

Comment:
Is there anything we can do do read the @doc attributes of the docs in this module?

Reply:
The `Refs` module uses `Code.fetch_docs`, which requires the bytecode to be written to disk ([reference](https://hexdocs.pm/elixir/writing-documentation.html#code-fetch_docs-1)) and that's not the case for in-memory modules. I don't know if there's any way around that.